cpp/src/slice2cs/Gen.cpp

```diff
@@ -2360,6 +2358,9 @@ Slice::Gen::ProxyVisitor::visitOperation(const OperationPtr& operation)
     string cancel = getEscapedParamName(operation, "cancel");
     string progress = getEscapedParamName(operation, "progress");
 
+    string requestProperty = "_iceI_" + opName + "Request";
```

Comment:
Since it's a property (not a field), and even though it's private, I think an `IceI_` prefix is more appropriate. Leading underscores are for private fields.